env_modules.py

```diff
@@ -50,6 +50,29 @@ def _module_old(*args):
         exec(output)
         # pylint: enable=exec-used
 
+    @staticmethod
+    def _setup_old(default_func):
+        """Ensure MODULEPATH is set for invoking modulecmd"""
+        path = os.environ.get("MODULEPATH", None)
+        if path:
+            return _env_module._module_old
+        mhome = os.environ.get("MODULESHOME", None)
+        if mhome is None:
+            print "no module home"
```

Comment:
(pylint-syntax-error) Missing parentheses in call to 'print'. Did you mean print("no module home")?